test/api/news.test.ts

```diff
@@ -0,0 +1,95 @@
+import { getNewsHandler, postNewsHandler} from '../../pages/api/news';
+import httpMocks, { Body } from 'node-mocks-http';
+import prisma from '../../prisma/prisma';
+import { News } from '@prisma/client';
+import { NewsWrite } from '../../lib/api/createNewsRequest.dto';
+import { BadRequestError } from '../../lib/utils/errors/badRequestError';
+
+const createNews0:NewsWrite = {
+  title: 'First news',
+  content: ':)'
+};
+
+const createNewsBad:unknown = {
+  title: 'i\'m missing content field!'
+}
+
+const news0:News = {
+  id: 0,
+  title: 'First news',
+  content: ':)',
+  createdAt: new Date('1997-08-17')
+};
+
+const news1 = {
```

Comment:
We should make this type News, aka `const news1: News = {`

Reply:
We should also test that having an empty string for title or content is invalid. We should also test that having no title is invalid